app/views/jst/developer_key.handlebars

```diff
@@ -42,9 +42,19 @@
     </td>
     <td>
         <div>
+          {{#t "access_token_count"}}Access Token Count: {{access_token_count}}{{/t}}
```

Comment:
Oh okay cool.  I'm glad we have some external facing docs around that 😅.

It's such a toss up for me.  Part of me says kill the existing keys so we are closer to having none, but that does add a bit more to the scope of your change.  Part of me says not to worry about it and if those strings ever change then we'll kill it.  I don't _think_ we'll reject the PR either way.